bin/add_frontend_server.rb

```diff
@@ -241,10 +272,12 @@ def execute_ssh_on_channel(ssh, command, exit_error_string)
 cmd = "ssh gateway.code.org -t knife bootstrap #{private_dns_name} -x ubuntu --sudo -E #{environment} -N #{instance_name} -r role[#{role}]"
```

Comment:
We don't output the result except when things fail (which is very rare) so this turns out not to be so important. 

It also appears that chef doesn't flush it's output very frequently so even if you try this it doesn't work every well.